internal/services/mssql/helper/sku.go

```diff
@@ -0,0 +1,25 @@
+package helper
+
+import (
+	"strings"
+)
+
+// CompareDatabaseSkuServiceTiers returns true if sku1 has a higher service tier than sku2
+func CompareDatabaseSkuServiceTiers(sku1, sku2 string) bool {
+	// These are intentionally short so that both forms can be matched, e.g. "S1" or "Standard"
+	order := []string{
+		"", "B", "S", "P",
+	}
```

Comment:
I'm still doing some more testing to add the remaining known SKU families and determine their order, and will add them in along with test coverage.

Reply:
This is done